src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTagger.cs

```diff
@@ -19,56 +24,118 @@ internal sealed class InlineParameterNameHintsTagger : ITagger<IntraTextAdornmen
     {
         private readonly ITagAggregator<InlineParameterNameHintDataTag> _tagAggregator;
         private readonly ITextBuffer _buffer;
+        private readonly ITextView _textView;
+
+        /// <summary>
+        /// _cache stores the parameter hint tags in a global location 
```

Comment:
```suggestion
        /// Stores the parameter hint tags in a global location 
```

No reason to duplicate the name; this comment applies this so that part is obvious.